src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/casenotes/CaseNotesReconciliationService.kt

```diff
@@ -177,40 +177,40 @@ class CaseNotesReconciliationService(
 
   private val truncatedToSecondsFormatter = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss")
 
-  private fun CaseNote.transformFromDps() = CommonCaseNoteFields(
-    text.truncateToFixNomisMaxLength(),
-    type,
-    subType,
-    occurrenceDateTime.format(truncatedToSecondsFormatter),
-    creationDateTime.format(truncatedToSecondsFormatter),
-    authorUsername,
-    null,
-    amendments.map { a ->
+  private fun CaseNote.transformFromDps() = ComparisonCaseNote(
+    id = caseNoteId,
```

Comment:
add in the case note id so that we can easily go to DPS to get the case note.  At present we have to get all the case notes for a prisoner and then look for the one we need